app/src/main/java/com/nextcloud/talk/controllers/CallNotificationController.java

```diff
@@ -343,11 +343,15 @@ private void runAllThings() {
         showAnswerControls();
     }
 
-    @SuppressLint("LongLogTag")
+    @SuppressLint({"LongLogTag"})
     @Override
     protected void onViewBound(@NonNull View view) {
         super.onViewBound(view);
 
+        String callDescriptionWithoutTypeInfo =
+                getResources().getString(R.string.nc_app_name) + " " + getResources().getString(R.string.nc_call_unknown);
+        incomingCallVoiceOrVideoTextView.setText(callDescriptionWithoutTypeInfo);
```

Comment:
String concatenation are normally done as a separate string with "%s1 and %s2" so that left to right languages work.

Reply:
but here i combine two translated strings. it would be possible to make another string that has the value "%1$s %2$s" so for other languages it could be changed to "%2$s %1$s", but i don't think anyone will understand that?
or do i get it wrong?

Reply:
True, this is hard to understand on translator side.
Maybe then have "%s call", where %s is app name?

Reply:
ah sure 🤦‍♂️ 😆